Client/mods/deathmatch/logic/CRadarMap.cpp

```diff
@@ -114,6 +122,24 @@ CRadarMap::~CRadarMap()
     // Don't need to delete the help texts as those are destroyed by the display manager
 }
 
+void CRadarMap::SetMapImage(std::uint32_t imageIndex)
+{
+    std::uint32_t width, height;
+    if (imageIndex < RADAR_IMAGE_SIZES.size())
+        width = height = RADAR_IMAGE_SIZES[imageIndex];
+    else
+        width = height = RADAR_IMAGE_SIZES[0];
+
+    SString fileName("MTA\\cgui\\images\\radar_%d.png", width);
+
+    if (m_pRadarImage)
+        SAFE_RELEASE(m_pRadarImage);
+
+    m_pRadarImage = g_pCore->GetGraphics()->GetRenderItemManager()->CreateTexture(CalcMTASAPath(fileName), nullptr, false, width, height, RFORMAT_DXT1);
```

Comment:
Can this function throw a error or return `nullptr` in cases if errors? 
What happens if a client is unable to load a texture? Do we need fallback?

Reply:
Client never failed to load the map texture in years of MTA. There was never a check. Idk if it would happen for bigger textures, hmm...

Reply:
Did you check it?